lib/Current.dart

```diff
@@ -0,0 +1,706 @@
+import 'dart:ui';
+import 'Login.dart';
+import 'package:flutter/material.dart';
+import 'size_config.dart';
+
+class Current extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    SizeConfig().init(context);
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(
+          'Schedule',
+          style:
+              TextStyle(color: Colors.white, fontSize: 20, fontFamily: 'SFPro'),
+        ),
+        backgroundColor: Colors.black,
+        titleSpacing: 0.0,
+        iconTheme: IconThemeData(color: Colors.white),
+        automaticallyImplyLeading: true,
+        leading: IconButton(
+          icon: Icon(Icons.arrow_back_ios),
+          onPressed: () => Navigator.pop(context, false),
+        ),
+      ),
+      body: Column(
```

Comment:
Refactor parts of the body that are hard to read or that are used more than once into separate widgets. Also remember that widgets can have arguments, allowing us to factor out even more.